app/configurator/configurator-state/reducer.spec.tsx

```diff
@@ -705,9 +705,11 @@ describe("colorMapping", () => {
       chartConfig.fields.color.type === "single"
         ? chartConfig.fields.color.color
         : chartConfig.fields.color.colorMapping
-    ).toEqual({
-      orange: "rgb(255, 153, 0)",
-    });
+    ).toEqual(
+      chartConfig.fields.color.type === "single"
+        ? "rgb(255, 153, 0)"
+        : { orange: "rgb(255, 153, 0)" }
+    );
   });
```

Comment:
@noahonyejese I think the problem with this test was that we should be sure on what we expect to appear here. It looks like here we update a chart field (add a new segment), so every test down the line should make sure that we get segment palette types, like

```typescript
expect(chartConfig.fields.color.colorMapping).toEqual({ orange: "rgb(255, 153, 0)" });
```

It would be cool to add another test, or remove a segment field in this test, and expect the color field to be of `single` type, with rgb color value. Does it make sense? 😄

Reply:
Okay I see, I will change this